fonduer/utils/data_model_utils/structural.py

```diff
@@ -0,0 +1,198 @@
+#############################
+# Structural modality helpers
+#############################
+
+from builtins import str
+
+import numpy as np
+from lxml import etree
+from lxml.html import fromstring
+
+from fonduer.utils.data_model_utils.utils import _to_span
+
+
+def get_tag(mention):
+    """Return the HTML tag of the Mention.
+
+    If a candidate is passed in, only the tag of its first Mention is returned.
```

Comment:
candidate?

Reply:
I see. Should we return a list of tags? thoughts?

Reply:
I think it is ok as is.